data/json/items/gun/shot.json

```diff
@@ -677,8 +677,8 @@
     "id": "tavor_12",
     "copy-from": "shotgun_base",
     "type": "GUN",
-    "name": { "str": "gas-operated bullpup shotgun" },
-    "description": "Looking more like some form of sci-fi prop gun than an actual firearm, this weapon uses a system of three tube magazines to field 15 shotshells within a comparatively compact design.  Each tube has to be loaded separately, but this offers the option of loading different ammunition for different situations.  In addition, as the weapon’s trigger assembly is situated forwards of the chamber and magazines in what is termed as a 'bullpup' design, the firearm’s notably more compact than its peers despite having a comparatively high capacity when compared against more classically designed shotguns.",
+    "name": { "str": "auto-loading bullpup shotgun" },
+    "description": "Looking more like some form of sci-fi prop gun than an actual firearm, this weapon uses a system of triple magazines to field 15 12-gauge shotgun shells within a comparatively compact design.  Each tube has to be loaded separately, but this offers the option of loading different ammunition for different situations.  As the weapon’s trigger assembly is situated forwards of the chamber and magazines in what is termed as a 'bullpup' design, the firearm’s notably more compact than its peers despite having a higher capacity when compared against more classically designed shotguns.",
```

Comment:
"Triple magazine" says "one conjoined object made of three individual magazines" to me